db/migrate/20170529020145_create_images.rb

```diff
@@ -1,10 +0,0 @@
-class CreateImages < ActiveRecord::Migration[5.0]
```

Comment:
Normally you wouldn't ever want to delete a migration, and instead would delete the existing table or migrate the data in the new migration. I think it's _probably_ OK in this particular case since it's very unlikely anybody's using this - but it might not hurt to practice doing it the "correct" way anyway?

Reply:
I guess not? I was expecting to see a migration that removed the table or (even better for a "real" case but probably overkill here) that migrated data from the old table to the new one